packages/jasmine/BUILD.bazel

```diff
@@ -81,7 +84,7 @@ pkg_npm(
     ],
     build_file_content = "",
     substitutions = {
-        "//packages/jasmine:jasmine_runner.js": "//:node_modules/@bazel/jasmine/jasmine_runner.js",
+        "//packages/jasmine/private:jasmine_runner_test.bzl": "//@bazel/jasmine:private.bzl",
```

Comment:
it's a bit non-symmetric. Maybe the RHS should be `//@bazel/jasmine/private:index.bzl` (we have that degree of freedom for the name of the generated file that I bike-shedded earlier?)

Having transformations be fewer steps away from "identity transformation" is less complex overall

Reply:
Agreed.